vm/src/stdlib/bisect.rs

```diff
@@ -0,0 +1,182 @@
+pub(crate) use _bisect::make_module;
+
+#[pymodule]
+mod _bisect {
+    use std::convert::TryFrom;
+
+    use crate::function::OptionalArg;
+    use crate::slots::PyComparisonOp::Lt;
+    use crate::vm::VirtualMachine;
+    use crate::{ItemProtocol, PyObjectRef, PyResult};
+    use num_traits::ToPrimitive;
+
+    #[derive(FromArgs)]
+    struct BisectArgs {
+        #[pyarg(any, name = "a")]
+        a: PyObjectRef,
+        #[pyarg(any, name = "x")]
+        x: PyObjectRef,
+        #[pyarg(any, optional, name = "lo")]
+        lo: OptionalArg<PyObjectRef>,
+        #[pyarg(any, optional, name = "hi")]
+        hi: OptionalArg<PyObjectRef>,
+    }
+
+    // Handles objects that implement __index__ and makes sure index fits in needed isize.
+    #[inline]
+    fn handle_default(
+        arg: OptionalArg<PyObjectRef>,
+        default: Option<isize>,
```

Comment:
I think it's better to just return `None` in the `Missing` case, and then you can choose what you want to do with the option and avoid `.unwrap()`s - e.g. instead of passing `handle_default(Some(1)).unwrap()`, you can do `handle_default()?.unwrap_or(1)`